dashboard/frontend/actions/index.js

```diff
@@ -72,10 +75,19 @@ const mountSockets = (store) => {
     socket.on('run-event', (data) => {
         const run = store.find(data.id);
         run.addEvent(data);
+
+        if(data.err) {
+            // mark run as errored
+            run.setInterrupted();
+        }
     });
 
     socket.on('run-stats', (data) => {
         const run = store.find(data.id);
+
+        // ignore the run stats if it is paused
+        if(run.isPaused()) return;
```

Comment:
Why do we want to ignore the run stats if the run is paused?

Reply:
Wouldn't that be ideal? I mean the run should not be doing anything right? 
Previously it used to keep updating the stats even when paused